docker/puppetdb/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:16.04
+
+ARG vcs_ref
+ARG build_date
+ENV PUPPETDB_VERSION="5.2.2"
+ENV PUPPET_AGENT_VERSION="5.5.1"
+ENV DUMB_INIT_VERSION="1.2.1"
+ENV UBUNTU_CODENAME="xenial"
+ENV PUPPETDB_DATABASE_CONNECTION="//postgres:5432/puppetdb"
+ENV PUPPETDB_USER=puppetdb
+ENV PUPPETDB_PASSWORD=puppetdb
+ENV PUPPETDB_JAVA_ARGS="-Djava.net.preferIPv4Stack=true -Xms256m -Xmx256m"
+ENV PATH=/opt/puppetlabs/server/bin:/opt/puppetlabs/puppet/bin:/opt/puppetlabs/bin:$PATH
+
+LABEL org.label-schema.maintainer="Puppet Release Team <release@puppet.com>" \
+      org.label-schema.vendor="Puppet" \
+      org.label-schema.url="https://github.com/puppetlabs/puppetdb" \
+      org.label-schema.name="PuppetDB" \
+      org.label-schema.license="Apache-2.0" \
+      org.label-schema.version=$PUPPETDB_VERSION \
+      org.label-schema.vcs-url="https://github.com/puppetlabs/puppetdb" \
+      org.label-schema.vcs-ref=$vcs_ref \
+      org.label-schema.build-date=$build_date \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.dockerfile="/Dockerfile"
+
+RUN apt-get update && \
+    apt-get install -y wget netcat lsb-release && \
+    wget https://apt.puppetlabs.com/puppet5-release-"$UBUNTU_CODENAME".deb && \
+    wget https://github.com/Yelp/dumb-init/releases/download/v"$DUMB_INIT_VERSION"/dumb-init_"$DUMB_INIT_VERSION"_amd64.deb && \
```

Comment:
Wonder if it might eventually be worthwhile to have "trust on first use" style checksum validation in in cases where we don't have more substantial authentication, i.e. like this: https://github.com/puppetlabs/puppetdb/pull/2488/commits/2d97bac78fcde16bec12bbe657e69d9922f3a0f8#diff-a1cdf43481aa7800ceaca164deaf8e3bR39

Reply:
This should be ticketed for future improvement, but is out of scope for the initial work

Reply:
Oh, sure - not critical right now, just thought it might be worth consideration.

Reply:
https://tickets.puppetlabs.com/browse/CPR-588